### ISSUE TITLE
ctac_to_mu_values problem if BUILD_EXECUTABLES=OFF

### ISSUE THREAD

**Issue by KrisThielemans** [UCL/STIR#515]
error is
```
CMake Error at src/utilities/CMakeLists.txt:90 (target_link_libraries):
  Cannot specify link libraries for target "ctac_to_mu_values" which is not
  built by this project.
```

**Comment by rijobro** [comment]
Easy fix, I'll do tomorrow

**Comment by KrisThielemans** [comment]
I'm halfway. seems we'll need a Travis job with `BUILD_EXECUTABLES=OFF`

**Comment by rijobro** [comment]
Ok, you figured out how? Should be the last `if` statement of that cmake file, that should need an `AND BUILD_EXECUTABLES`

**Comment by KrisThielemans** [comment]
closed with #516
